src/queryBuilder/Rule.js

```diff
@@ -1,23 +1,21 @@
 import React from 'react'
-import { Component, lenservable, hover } from '../mobx-react-utils'
+import { Component, lenservable, hover } from '../utils/mobx-react-utils'
 import styles from '../styles'
 import Indentable from './preview/Indentable'
 import FilterContents from './FilterContents'
 import FilterDragSource from './DragDrop/FilterDragSource'
-import { oppositeJoin } from '../searchUtils'
+import { oppositeJoin } from '../utils/search'
 
-let Rule = args => {
-  let {
-    state,
-    node,
-    tree,
-    root,
-    fields,
-    connectDragSource,
-    // connectDragPreview,
-    isDragging,
-  } = args
-  return connectDragSource(
+let Rule = ({
+  state,
+  node,
+  tree,
+  root,
+  connectDragSource,
+  // connectDragPreview,
```

Comment:
Is this still relevant?

Reply:
Nope, that's from when I was still experimenting with the react dnd api :D